core/pom.xml

```diff
@@ -152,6 +157,7 @@
             org.wso2.carbon.kernel.*; version="${carbon.kernel.package.export.version}",
         </export.package>
         <import.package>
+            org.wso2.carbon.kerenl.utils.*;version="${org.eclipse.osgi.util.package.import.version.range}",
```

Comment:
The import version should be the Carbon util version range which should be 5.0.0 to 6.0.0